### ISSUE TITLE
Error when starting without cluster configuration

### ISSUE THREAD

**Issue by crolopez** [wazuh/wazuh#1854]
## Bug report

**OS**
_CentOS 6_

**Wazuh version**
_3.7.1-3720_

**Log sample**

```
[root@centos6-agent src]# 2018/11/12 10:37:19 ossec-remoted: ERROR: Cannot read cluster config Success
```

Bug description
----------------

Wazuh gives an error (twice) if it starts without the cluster block. Although it is included in the template, if you use an old ossec.conf this message will appear from 3.7.0.

I think that, for esthetic reasons, our message should be separated from the error message.

Now:
https://github.com/wazuh/wazuh/blob/8df254c7e9cc2775b79f86fe14b1f6dfc05ff51e/src/remoted/main.c#L139

Proposal:
```c 
merror("Cannot read cluster config: %s (%d).", strerror(errno), errno);
```

**Comment by crolopez** [comment]
Related to https://github.com/wazuh/wazuh/issues/1868.

**Comment by crolopez** [comment]
The manage_agents tool does not work for the same reason.

> 2018/11/19 10:17:27 manage_agents: ERROR: Invalid option at cluster configuration

Reported by @MiguelCasaresRobles.

**Comment by DaveVG1** [comment]
The agent_control tool also doesn't work

**Comment by DaveVG1** [comment]
Both errors were fixed in the previous pull request #1900

**Comment by vikman90** [comment]
This issue depends on #1178. We must decide the default behavior (enabled or disabled) in order to fix this issue properly.
